react/week2/todo-app/src/Timer.js

```diff
@@ -0,0 +1,17 @@
+import React, {useState} from 'react';
+
+const Timer = () => {
+const [timeValue, setTimeValue] = useState(0);
+
+setInterval(() => {
+    setTimeValue(timeValue+1)
+}, 1000);
```

Comment:
It's important to put this logic inside when the **component mounts** and to **cleanup** the counting/interval when the timer is unmounted otherwise it may cause unwanted side effects like re-rendering each time the To Do list is updated and making it add inconsistently

(Hint: useEffect on mount with a cleanup)

Reply:
done